Cargo.lock

```diff
@@ -830,6 +846,16 @@ dependencies = [
  "libc 0.2.20 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
```

Comment:
why do we need `rand` now?

Reply:
rand is already in the dependency tree, this is just a dupe because some dependencies haven't updated yet (I've started doing what's needed for that)